### ISSUE TITLE
jlpm run build:all by default?

### ISSUE THREAD

**Issue by jtpio** [jupyterlab/extension-cookiecutter-ts#87]
Currently in 3.0 the default `build` script doesn't execute the `jupyter labextension build` command:

https://github.com/jupyterlab/extension-cookiecutter-ts/blob/e1a6cd634939eac14e7ccf956ea8b593466bc78e/%7B%7Bcookiecutter.python_name%7D%7D/package.json#L28-L31

Maybe `jlpm run build` could actually replace `jlpm run build:all`, so the workflow and commands would stay the same when switching to new extension system?

With the scripts updated to:

```json
 "build": "jlpm run build:lib && jlpm run build:labextension", 
 "build:labextension": "jupyter labextension build .", 
 "build:lib": "tsc", 
```

**Comment by jtpio** [comment]
So it also matches the `watch` scripts:

https://github.com/jupyterlab/extension-cookiecutter-ts/blob/e1a6cd634939eac14e7ccf956ea8b593466bc78e/%7B%7Bcookiecutter.python_name%7D%7D/package.json#L40-L42

**Comment by jasongrout** [comment]
Fixed in #88
